app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<div>
+  <h1>Hello admin</h1>
+  <h2>Displaying Square63 clients</h2>
+  <table class="table table-dark">
+    <thead>
+      <tr>
+        <th >#</th>
+        <th > Name </th>
+        <th >Role</th>
+        <th >Email</th>
+        <th >Action</th>
+      </tr>
+    </thead>
+    <tbody>
+    	<% @users.each do |u| %>
+        <% if u.user_role == User::ROLES[:client] %>
+          <tr>
+  	        <th scope="row">
+              <%= u.id %>
+            </th>
+  	        <td>
+              <%= u.firstname %>
+              <%= u.lastname %>
+            </td>
+            <td>
+              <%= u.email %>
+            </td>
+            <td>
+              <%= u.user_role %>
+            </td>
+  	        <td>
+  	      	  <%= link_to "Show ", admin_user_path(u), class: 'btn btn-primary' %> 
```

Comment:
Can we use `[:admin, user]` here?
Same for below two links with slight modifications